#HYPOTHESIS TESTING

**Question:**



*  Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.

**Objective:**


*  To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing




#Define Hypothesis:

**Null Hypothesis (Ho):**  The weekly operating costs are as per model.



**Alternative Hypothesis:**  The weekly operating costs are higher than the model.

#Given Data

 Sample Mean weekly cost      : 3050

 Theoretical mean weekly cost accordjing to the cost model   : $1000+$5X=1000+5*600  =4000    (X=600)

 standard deviation : 5*25=125 units

 sample size(n)     : 25 resturants



In [ ]:
import math #importing required libraries
import scipy.stats as stats
import numpy as np
import pandas as pd



In [ ]:
sample_mean=3050 #given data
population_mean=4000
Sample_standard_deviation=125
sample_size=25
alpha=0.05

#Test Statistics (Z)

In [ ]:
#t=(sample mean-Population mean)/Std Error
t=((3050-4000)/(125/np.sqrt(25))) # calculating the test statistic
print(t)


-38.0


#Critical Value

In [ ]:
alpha=0.05 #critical value for alpha is 0.05 in one tailed test.
critical_value=stats.norm.ppf(1 - alpha) # calculate the critical value
critical_value

1.6448536269514722

#Decision :

test statistic (Z) is -38,that is much lesser than the critical value 1.645. Therefore we faill to reject the null hypothesis.
```



In [ ]:
if t > critical_value:
    print("Reject the null hypothesis")
else:
    print("Fail to reject the null hypothesis")

Fail to reject the null hypothesis


#Conclusion :
Based on the hypothesis test, there is no enough evidence to support the claim that the observed weekly operating costs are higher than the theoretical costs predicted by the cost mode.
